__tests__/app.test.js

```diff
@@ -62,6 +62,24 @@ describe("GET", ()=>{
                 })
         })
     })
+    describe("/api/users", ()=>{
+        it("should return status: 200, and an array of objects containing specific properties",()=>{
+            return request(app)
+                .get("/api/users")
+                .expect(200)
+                .then(({body})=>{
+                    body.users.forEach((user)=>{
```

Comment:
we need a length check here to make sure that we aren't doing this forEach on an empty array. An empty array will pass this test automatically because the expect statement will never fail on an empty array